pkg/resource/lateinit.go

```diff
@@ -42,6 +42,7 @@ const (
 // GenericLateInitializer performs late-initialization of a Terraformed resource.
 type GenericLateInitializer struct {
 	valueFilters []ValueFilter
+	nameFilters  []NameFilter
```

Comment:
nitpick: I'd prefer having a single `[]Filter` to do all filtering operations but this is also perfectly fine.